AK/Slugify.cpp

```diff
@@ -9,13 +9,12 @@
 #include <AK/StringView.h>
 
 namespace AK {
-ErrorOr<String> slugify(String const& _raw, char const glue)
+ErrorOr<String> slugify(String const& input, char const glue)
```

Comment:
Please combine these changes into the commit that added this code, (which is in this PR,) instead of separately.

Reply:
Done